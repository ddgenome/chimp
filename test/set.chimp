## gas-phase only mechanism
mechanism "set.mech"
## parameter input
parameter "set.par"
## gas-phase task
task "set.task"
mechanism "set.comp.mech"
## parameter input
parameter "set.par"
## gas-phase task
task "set.task"
