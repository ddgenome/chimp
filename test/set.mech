# a simple gas-phase mechanism using the species set feature
D -> k_arrhenius(A2f, E2f) <- k_arrhenius(A2r, E2r) E;
A + [B, C] -> k_arrhenius(A1f, E1f) <- k_arrhenius(A1r, E1r) D;
