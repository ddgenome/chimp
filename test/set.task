# -*- text -*-
# test task input
begin model set
  output "set.out"
  begin integrator kmc
    scale 1.0e14
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
	p[B] = 1.0e4		# Pa
	p[C] = 1.0e4		# Pa
      end quantity
      begin output
	(1.0e0 1.0e1)
      end output
      begin reactor batch
	temperature 3.0e2	# K
	pressure 1.2e5		# Pa
	volume 1.0e-5		# m^3
	rate_numerator moles
	rate_denominator volume
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
