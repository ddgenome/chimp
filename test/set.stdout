chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task set.par... completed; 0u 0s 0w
chimp: performing task set... completed; 2.13u 0.01s 2.13w
chimp: performing task set.par... completed; 0u 0s 0w
chimp: performing task set... completed; 2.09u 0s 2.1w
